perftestnotebook/transformer.py

```diff
@@ -131,3 +133,47 @@ def merge(self, sde):
 
         self.entry_number = 0
         return merged
+
+
+def get_transformers(filepath="perftestnotebook/customtransforms"):
```

Comment:
This function should be part of the PerftestNotebook class and the predefined_transformers should be a PerftestNotebook object attribute.